scenes.go

```diff
@@ -53,35 +58,46 @@ func initScene(sceneNum int) (err error) {
 		}
 		sceneFS = referenceframe.NewEmptySimpleFrameSystem("test")
 		sceneFS.AddFrame(scene.RobotFrame, sceneFS.World())
+
+		if scene.EndEffectorFrame != nil {
```

Comment:
If we want to do this, we should have a chat within the group about how we envision test suite evolving. It'll make a lot of sense to be robust to much more than single goal, single start, single arm, but I'd like to knock out all our requirements at once. I'm going to schedule something on Monday for us to chat real quick and I can implement the changes likely same day.

Reply:
Ok sounds good.  In the meantime if you would like to consider my comment out of scope for the moment I'm good to push this off until a later time, it will be a relatively easy thing to change later.

Reply:
I'm just going to hold the PR altogether. It was pretty easy to make the change present in this commit and I can't imagine what we agree on is more than an extra day of work on top. We'll get a nice batch of improvements altogether and we can be truly ready for more scenes.

Reply:
Updated now, frames are handled within the scene config by a local Frame System. Also easy to make sure frames are added to the correct parents than handling that at the `initScene` level